### PR TITLE
Add and  Remove Tasks

### PR DESCRIPTION
**Pull request by DammyShittu**
> This is the final milestone of the Todo List app.

In this pull request, 
- Linters were properly set up
- Webpack was used
- Files were created as modules 

Functionalities were added to;
- Add Todo Task
- Edit Todo Task
- Remove Todo Task
- Clear Completed Tasks
- Indexes were updated after deleting a task to represent the current list order.
- Preserve Data In Local Storage

### CODE REVIEW

File: src/clearCompleted.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import showTasks from './showTasks.js';
+import { setLocalStorage } from './setLocalStorage.js';
+import getTasksFromLocalStorage from './getTasks.js';
+// eslint-disable-next-line import/no-cycle
+import setIndex from './index.js';
+// Remove Completed Tasks
+const removeCompletedTasks = () => {
+  let tasks = getTasksFromLocalStorage();
+  // let i = 1;
```
Comment (hemant-soni-vst-au4) [inline src/clearCompleted.js:9]: - [ ] Kindly try to delete the commented code. Following the best practices, we should not push commented codes and comments.

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @DammyShittu, 

Great work, you met all the requirements of this milestone. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to improve the getting started section in the README file. There are some missing steps to run the project locally like `npm install`, `npm run build` and `npm start`. Kindly add the instruction to run the project locally.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/clearCompleted.js (hunk 1 above)
Comment (DammyShittu) [reply src/clearCompleted.js:9]: Thank you very much @hemant-soni-vst-au4.
I will add the steps to run the project locally in my README file and also delete the commented code.

Comment (DammyShittu) [reply src/clearCompleted.js:9]: Hi Hemant,
I have implemented the changes you suggested.
Thanks 😁

**Pull request merged**